app/src/main/java/com/madonasyombua/growwithgoogleteamproject/MainActivity.java

```diff
@@ -184,18 +188,47 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
 
             case R.id.about:
                 // Take user to about screen
+
                 return false;
             case R.id.help:
                 // Take user to help screen
                 return false;
             case R.id.logout:
                 // logout
+                final AlertDialog.Builder builder = new AlertDialog.Builder(this);
+                builder.setMessage("Are you sure you want to logout?");
+                builder.setCancelable(true);
+
+                builder.setPositiveButton("Yes", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialogInterface, int i) {
+                        //log out the user and launch LoginActivity
+                        Toast.makeText(MainActivity.this, "Inside Yes Dialog", Toast.LENGTH_LONG).show();
+                        MainActivity.this.finish();
+                        Intent intent = new Intent(MainActivity.this, LoginActivity.class);
+                        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+                        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
+                        intent.addFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION);
+                        startActivity(intent);
+                        finish();
+                        System.exit(0);
```

Comment:
When a user logs out, it is not reasonable to kill the app rather log them out and send them to sign in fragment.

Reply:
Noted will notify @aymkhan  and can also help her 👍

Reply:
I see.

Reply:
also finish() is enough to quit app, but as @CleverChuk we have just to redirect them to login activity, also we have to update login status in SharedPreference like we do for login.